<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txt_output">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>10</y>
      <width>324</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLineEdit" name="edit_code">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>종목코드</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_code">
       <property name="text">
        <string>종목추가</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>계좌번호</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_get_balance">
       <property name="text">
        <string>잔고조회</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_condition_refresh">
       <property name="toolTip">
        <string>test</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>조건식 refresh</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btn_print_my_data">
       <property name="text">
        <string>my_data</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn_register_real">
       <property name="text">
        <string>실시간등록</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_register_real_all">
       <property name="text">
        <string>보유실시간</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="combo_account"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="table_current">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>771</width>
      <height>271</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>10</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="table_condition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>521</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>4</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTextEdit" name="txt_balance">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_code_del">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>190</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>선택종목 삭제</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>250</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>매도전략</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="combo_buy"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_sell"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>매수전략</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_buy_strategy_add">
       <property name="text">
        <string>선택종목에 매수전략 추가</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_sell_strategy_add">
       <property name="text">
        <string>선택종목에 매도전략 추가</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>110</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>선택종목</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPyStock">
    <property name="title">
     <string>PyStock</string>
    </property>
   </widget>
   <addaction name="menuPyStock"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_condition_refresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_condition_refresh_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_get_balance</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_balance_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_print_my_data</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_print_my_data_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_code</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_code_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_register_real</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_register_real_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_register_real_all</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_register_real_all_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_account</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_account_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_code_del</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_code_del_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>831</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>894</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_buy_strategy_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_buy_strategy_add_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>830</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sell_strategy_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_sell_strategy_add_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1038</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_condition_refresh_btn_clicked()</slot>
  <slot>on_condition_result_btn_clicked()</slot>
  <slot>on_balance_btn_clicked()</slot>
  <slot>on_print_my_data_btn_clicked()</slot>
  <slot>on_code_btn_clicked()</slot>
  <slot>on_register_real_btn_clicked()</slot>
  <slot>on_register_real_all_btn_clicked()</slot>
  <slot>on_account_changed(QString)</slot>
  <slot>on_code_del_btn_clicked()</slot>
  <slot>on_buy_strategy_add_btn_clicked()</slot>
  <slot>on_sell_strategy_add_btn_clicked()</slot>
 </slots>
</ui>
